labelbox/client.py

```diff
@@ -1786,3 +1789,104 @@ def get_batch(self, project_id: str, batch_id: str) -> Entity.Batch:
             experimental=True)["project"]["batches"]["nodes"][0]
 
         return Entity.Batch(self, project_id, batch)
+
+    def send_to_annotate_from_catalog(
+            self,
+            batch_name: str,
+            data_rows: DataRowIdentifiers,
+            source_model_run_id: Optional[str],
+            source_project_id: Optional[str],
+            destination_project: str,
+            task_queue_id: Optional[str],
+            exclude_data_rows_in_project: bool = False,
```

Comment:
An idea: what if we encapsulated all of the "configuration" type parameters in an object? If we decide to introduce a new configuration in the future as a required param, we'd have to rearrange the order of parameters in a breaking manner. Instead, what if we had a SendToAnnotateParams class as a TypeDict (or pydantic model) similar to export v2 params and filters? The benefit of this would be:
1. Less parameters in the function signature
2. Can reuse the same config object for both send_to_annotate functions